CHANGELOG.md

```diff
@@ -1,9 +1,10 @@
 # Changelog
 
-## [1.19.1](https://github.com/brave/brave-ios/releases/tag/v1.19.1)
```

Comment:
Does this imply we're skipping `1.19.1` now? Or just merging the hotfix release notes?

Reply:
@kylehickinson I though `1.19.1` was blocked/rejected by Apple hence doing a `1.19.2`? Or are we planning on releasing both versions?

Reply:
As far as I am aware, we are planning on releasing both

Reply:
Dropping 1.19.1 is only a last resort in case we have to make code changes in order to pass the review.
Then we could think about merging both releases

Reply:
@srirambv sounds like we'll need to split these back into two for now and have an entry for `1.19.1` 👍 Apologies, my fault for suggesting to merge them as I thought we're skipping `1.19.1` and moving to `1.19.2`.

Reply:
split the notes for 1.19.1 and 1.19.2